packages/core/src/controllers/SwapController.ts

```diff
@@ -739,6 +755,9 @@ export const SwapController = {
         }
       })
       SwapController.resetState()
+      if (!isAuthConnector) {
+        RouterController.replace('Account')
+      }
```

Comment:
What happens if it's the auth connector?

Reply:
If it's auth connector, we are handing the success and error states on top with `RouterController.pushTransactionStack`, there is no need to call something here for auth connectors bc the way they are working is different.